app/routes/docs.tsx

```diff
@@ -81,6 +81,25 @@ export default function () {
                     ]}
                   />
                 </>,
+                useFeature('SCHEMA') ? (
+                  <>
+                    <NavLink key="producers" to="producers">
```

Comment:
The main Notices documentation has now been merged, so you should rebase to get producers under docs.notices.producers

Reply:
Ok, hopefully rebase would be easy.